spec/models/notification_spec.rb

```diff
@@ -341,6 +341,16 @@ def process_alerts(post)
 
       expect(Notification.count).to eq(2)
     end
+
+    it 'does not delete chat_message notifications' do
+      user = Fabricate(:user)
+      Notification.create!(read: false, user_id: user.id, topic_id: nil, post_number: nil, data: '[]',
+                           notification_type: Notification.types[:chat_mention])
+
+      expect {
+        Notification.ensure_consistency!
+      }.to change { Notification.count }.by(0)
```

Comment:
`to_not change { Notification.count }`

is a lot nicer imo

Reply:
nice!